.clang-format

```diff
@@ -57,7 +57,7 @@ BreakConstructorInitializersBeforeComma: false
 BreakConstructorInitializers: BeforeColon
 BreakAfterJavaFieldAnnotations: false
 BreakStringLiterals: true
-ColumnLimit:     0
+ColumnLimit:     100
```

Comment:
I did not add this specifically as we would need to change all the files.
Since this PR includes the change :+1:

Reply:
Seems that all the files are touched regardless of this restriction.  😄   But might as well bite the bullet on it!